src/Network/Request.php

```diff
@@ -158,6 +159,13 @@ class Request implements ArrayAccess
     protected $_session;
 
     /**
+     * Store the additional attributes attached to the request.
+     *
+     * @var array
+     */
+    protected $attributes = [];
```

Comment:
Agreed! I always liked the convention for the 'magic' features it allows but given that all other PHP code out there is sticking to no underscores in general for differentiating properties, it will make things easier to read in the longer term.

So, from here on, we should submit PRs following this new no underscore convention?